app/presenters/dashboard_presenter.rb

```diff
@@ -0,0 +1,27 @@
+class DashboardPresenter
```

Comment:
This is great. I've definitely gotten dinged for having too many instance variables crammed inside a controller method (as in lines 4-12 in the dashboard controller above), so using a presenter is a good move.